### ISSUE TITLE
NERC parameter usage vocabulary - test 

### ISSUE THREAD

**Issue by meganrwong** [ANZSoilData/def-au-scma#47]
Questions relating to the trial of the the observable properties as measured and reported by the scma soil chem methods handbook. Based on https://github.com/CSIRO-enviro-informatics/PUV-ont

The parameters that these methods are a measure of - some are easier to break out following NERC ontology pattern. Some are not as straight forward. Throwing some thoughts/questions up here to help get a better feel of how we may model - yet to get RDF examples up obviously. 

The methods up front of book throw curlier questions, should get more repetitive/easier with the other chapters here is hoping....

**Comment by meganrwong** [comment]
method http://anzsoil.org/def/au/scm/2A1
air-dried moisture content (PO1 equiv) would have sample preparation (S03 equiv) of 'air dried'. 

 is in contrast to http://anzsoil.org/def/au/scm/3A1
which reports 'Electrical conductivity (EC) at 25°C on an air-dry (40°C) basis'  
 looking at NERC example, it looks like 'on-air-dry (40 degC) basis' is a Measurement-Matrix relationship https://vocab.nerc.ac.uk/collection/S02/current/    - S02003 

looking for feedback on my interpretation here

**Comment by meganrwong** [comment]
Estimated soluble salts concentration, estimated as Electrical conductivity 

Property (S06 equiv) is 'Soil ionic strength' (or 'ionic strength' - if we were to be consistent w out ASLS modelling I'd say it is just ionic strength) 
Data processing method is 'estimated' 
(plus the method and related UoM, quantity kind)

thoughts?

**Comment by meganrwong** [comment]
'Estimated soluble salts concentration, estimated as Electrical conductivity '
is 
Property (S06 equiv) - Concentration of soluble salts
Data processing method (S05 equiv) is 'estimated as Electrical Conductivity  - I think!?!
Plus the related method, UoM and quantity kind) 

thoughts?

**Comment by meganrwong** [comment]
Parameter -  'Redox potential - field Eh values, retaining information on point-to-point variability'
http://anzsoil.org/def/au/scm/3D1

property: Redox potential
analytical method would also be http://anzsoil.org/def/au/scm/3D1
Data processing method  - 'retaining information on point-to-point variability' 

question - I'm a bit stumped on where 'field Eh values' would fit - it could be under a 2nd analytical method, but I'm leaning more toward it being a data processing method?

**Comment by meganrwong** [comment]
Parameter - 
pH (1:5 soil/water) on an air-dry basis
property - pH
analytical method - http://anzsoil.org/def/au/scm/4A1
measurement matrix relationship (I think!?) - 'in an air-dried basis
phase (S23 equiv) - 1:5 soil/water

**Comment by dr-shorthair** [comment]
Indeed, there is some inconsistency in the NVS vocabs. The granularity level of S06 and S03 varies. Do your best but don't stress too much. If in doubt: think about the use-case. I.e. how does factoring the parameter assist in discovery, and what terms would a user want to use in their search. There may be more than one answer.

**Comment by meganrwong** [comment]
Up to end of nitrogen chapter. Main places stuck to talk through - 
- Parameters are becoming very lengthy (same as methods) or too short (perhaps not unique enough for the different things they are a measure of)
- When there is more than one element measured eg 'Mineral N' 
- Ratios - how best to represent, when they are different forms of elements they measure, and different method combinations that measure them 
- Elements - where to put things like "total organic carbon", "Potentially mineralisable nitrogen (PMN)"  and 'total Nitrogen'  - are these the property or the element? NERC's S27 is elements, compounds, isotopes and mixtures. If they are a property rather than an element, perhaps they will look something like 'concentration of potentially mineralisable N' and ' proportion of total N', and the elements and compounds they measure are in our S27 equivalent (such as NH4, NO3)
